numpy/linalg/_linalg.py

```diff
@@ -1741,6 +1741,40 @@ def svd(a, full_matrices=True, compute_uv=True, hermitian=False):
         return s
 
 
+def _svdvals_dispatcher(x):
+    return (x,)
+
+
+@array_function_dispatch(_svdvals_dispatcher)
+def svdvals(x, /):
+    """
+    Returns the singular values of a matrix (or a stack of matrices) ``x``.
+    When x is a stack of matrices, the function must compute the singular
+    values for each matrix in the stack.
+
+    This function is Array API compatible.
+
+    Parameters
+    ----------
+    x : (..., M, N) array_like
+        Input array having shape (..., M, N) and whose innermost two
```

Comment:
Maybe "last two" instead of "innermost"?

Reply:
Sure - I changed it to "last two".